versavis/src/versavis_synchronizer.cpp

```diff
@@ -17,7 +17,7 @@ VersaVISSynchronizer::VersaVISSynchronizer(const ros::NodeHandle &nh,
                                            const ros::NodeHandle &nh_private)
     : nh_(nh), nh_private_(nh_private), image_transport_(nh),
       received_first_camera_info_(false), kMaxImageCandidateLength(10),
-      kMinSuccessfullConsecutiveMatches(4), kMaxImageDelayThreshold(0.1),
+      kMinSuccessfullConsecutiveMatches(4), kMaxImageDelayThreshold(0.5),
```

Comment:
This might depend on setup, but with such a high delay threshold, I would suggest lowering the initialization speed